src/ray/core_worker/core_worker.cc

```diff
@@ -1030,17 +996,18 @@ void CoreWorker::HandleGetObjectStatus(const rpc::GetObjectStatusRequest &reques
   if (task_manager_->IsTaskPending(object_id.TaskId())) {
     // Acquire a reference and retry. This prevents the object from being
     // evicted out from under us before we can start the get.
-    AddObjectIDReference(object_id);
+    // XXX: ???
```

Comment:
In principle we can remove this once we have full distributed ref counting, since there will be a borrower entry preventing the task from getting evicted. But we probably still need it for now in the serialized object id edge case.